### ISSUE TITLE
Missing creation of file 'top_5k_keywords.txt' and missing 'dice_stop_words.txt'

### ISSUE THREAD

**Issue by denisb411** [DiceTechJobs/ConceptualSearch#3]
Hello @simonhughes22 ,

I started using the third ipynb (Conceptual Search - 3. Create Word2Vec Model.ipynb) and realized that the file 'top_5k_keywords.txt' wasn't created on previous steps. Could you please provide a code for it or even an example of how is the 'top_5k_keywords.txt' structure?

Also, the stop_words file is missing. I think that it's a small file and can be easily provided.

BR,
Denis.

**Comment by simonhughes22** [comment]
@denisb411 I added some example files to the "./sample input files" folder and some more comments to notebook 3. These are small samples of our data, I cannot provide the full files unfortunately as that is our data. However, I would use the python scripts where possible instead, those are more clearly commented and share common config files, so you don't have to hunt down filepaths in notebooks to fix them all in multiple locations. The top_5k_keywords.txt is just a lucene format synonym file that I used to extract our top 5k search terms from the job descriptions. I have included python code that emulates the solr and ES synonym processing chains. This can also be used to filter the documents only to the recognized keywords, depending on how you want to approach this.
